.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,42 +0,0 @@
-_PR description_
-
-### Contents
-
-- _item_
-
-### Rationale
-
-_design decisions and extended information_
-
-
-<hr>
-
-## How to fill a PR description (remove this)
-
-Please give a concise description of your PR.
-
-The target readers could be future developers, reviewers, and auditors. By reading your description, they should easily understand the changes proposed in this pull request.
-
-MUST: Reference the issue to resolve
-
-### Single responsability
-
-Is RECOMMENDED to create single responsibility commits, but not mandatory.
-
-Anyway, you MUST enumerate the changes in a unitary way, e.g.
-
-```
-This PR contains:
-
-- Cleanup of xxxx, yyyy
-- Changed xxxx to yyyy in order to bla bla
-- Added xxxx function to ...
-- Refactored ....
-```
-
-### Design choices
-
-RECOMMENDED to:
-- What types of design choices did you face?
-- What decisions you have made?
-- Any valuable information that could help reviewers to think critically
```

Comment:
We extensively discussed how the PR description could best help the reviewing process. We concluded that it is most helpful when the PR author discloses more detail about the design choices. Can we add back this part?